src/d3a/models/area/__init__.py

```diff
@@ -241,7 +241,9 @@ def now(self) -> DateTime:
 
     @property
     def all_markets(self):
-        return list(self._markets.markets.values())
+        return [m for m in self._markets.markets.values()
```

Comment:
Just to clarify: Here, you omit the additional markets that are created due to market_count > 1?

Reply:
Yes